js/core/component.js

```diff
@@ -490,6 +456,14 @@ var Component = Class.inherit({
         return result;
     },
 
+    _getOptionSilent: function(name) {
```

Comment:
Maybe it should be `_getOptionByStealth`? I think, pairy methods should be named the same way.